src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -436,7 +436,14 @@ public void terminatedPod() throws Exception {
         r.waitForMessage("+ sleep", b);
         deletePods(cloud.connect(), getLabels(this, name), false);
         r.assertBuildStatus(Result.ABORTED, r.waitForCompletion(b));
-        r.waitForMessage(new ExecutorStepExecution.RemovedNodeCause().getShortDescription(), b);
+        r.waitForMessage("Container jnlp was terminated", b);
```

Comment:
https://github.com/jenkinsci/kubernetes-plugin/pull/772/files#diff-55ff61b56516ad2fa5dd36df80c3836cR213 intercepts the event before the node is actually removed.

Reply:
Mm, so making this test not exactly useless but not testing the original fix. OK.

Reply:
I was able to restore the behaviour by not interrupting the executors in the new listeners.